paddle/fluid/inference/api/analysis_predictor.cc

```diff
@@ -1934,64 +1921,6 @@ bool AnalysisPredictor::LoadProgramDesc() {
   return true;
 }
 
-bool AnalysisPredictor::LoadParameters() {
```

Comment:
why delete this funtion? And not delete it in .h ?

Reply:
> why delete this funtion? And not delete it in .h ?

Not used. It should also be deleted in .h.

Reply:
Keep it.